src/lib/analyses.test.ts

```diff
@@ -472,12 +471,9 @@ describe('getExperimentHealthIndicators', () => {
             "recommendation": "Experiment has been running for way too long. Stopping it now is highly recommended.",
             "severity": "Warning",
           },
-          "link": "https://github.com/Automattic/experimentation-platform/wiki/Experiment-Health#experiment-run-time",
+          "link": "https://fieldguide.automattic.com/the-experimentation-platform/experiment-health/#experiment-run-time",
           "name": "Experiment run time",
           "unit": "days",
-          "value": ${expectedExperimentRunTimeDays},
-        },
-      ]
-    `)
+          "value": `)
```

Comment:
This should fix the failing unit test:
```suggestion
          "value": ${experimentRunTimeDays},
        },
      ]
    `)
```

Reply:
Thanks for the suggestion! I ended up adding this back in https://github.com/Automattic/abacus/pull/729/commits/bc3a8a44ddbc89e5f25f409ee55a5f00118631da.